airlock/slack.py

```diff
@@ -3,9 +3,18 @@
 from .config import ORG_SLACK_CHANNELS
 
 
-def post_slack_update(org, update_url, issue_title, update_text):
+def post_slack_update(org, update_url, issue_title, update_text, github_error):
     channel = ORG_SLACK_CHANNELS.get(org, ORG_SLACK_CHANNELS["default"])
 
-    title_link = slack.link(update_url, f"{issue_title} has been updated")
-    message_text = f"{title_link}\n{update_text}"
+    if update_url is not None:
+        title = slack.link(update_url, f"{issue_title} has been updated")
+    else:
+        title = f"{issue_title} has been updated"
+    message_text = f"{title}\n{update_text}"
+
+    if github_error:
+        message_text += (
+            f"\nA GitHub error occurred, please contact tech-support: {github_error}"
```

Comment:
What can tech-support do here?

Reply:
Mostly just check what went wrong? I can remove this if we think it's not worth telling people to contact tech support. They probably will anyway, if they see an error reported

Reply:
OK.  No strong opinion from me.